frontends/qt/server/server.go

```diff
@@ -128,12 +128,16 @@ func serve(
 				C.free(unsafe.Pointer(cResponse))
 			},
 			pushNotify: func(msg string) {
-				C.pushNotify(pushNotificationsFn, C.CString(msg))
+				cMsg := C.CString(msg)
+				C.pushNotify(pushNotificationsFn, cMsg)
```

Comment:
just like with a mutex lock/unlock, i think it's "safer" to defer C.free:

```go
cMsg := C.CString(msg)
defer c.free(unsafe.Pointer(cMsg))
C.pushNotify(....)
```

i mean, everywhere. but this also works i guess.